dist/main.js

```diff
@@ -0,0 +1,32 @@
+const expenseMan = new ExpenseManager()
+const renderer = new Renderer()
+
+renderer.renderAdd()
+
+const addExpense = function() {
+    let expense = {
+        name: $("input:first-child").val(),
+        amount: $("#amount").val(),
+        group: $("select").val(),
+        date: $("#date-select").val() || moment().format("YYYY-MM-DD")
+    }
+    console.log(expense)
+    expenseMan.addExpense(expense)
+}
+
+const renderExpenses = async function(group, startDate = 0, endDate = 0) {
+    !startDate ? startDate = moment(new Date(1971, 0, 1, 0, 0, 0, 0)).format("YYYY-MM-DD") : startDate = moment(startDate).format("YYYY-MM-DD")
```

Comment:
better do it like this (generally we don't want to "ask" negatively but preffer to ask positively + have the ternary inside):
`startDate = moment(startDate ? startDate : new Date(1971, 0, 1, 0, 0, 0, 0)).format("YYYY-MM-DD")`

also something similar to the next line of code and to the `start` variable in the `dateRange` function.